modules/apps/commerce/commerce-product-content-search-web/src/main/resources/META-INF/resources/option_facets/configuration.jsp

```diff
@@ -0,0 +1,62 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+CPOptionFacetPortletPreferences cpOptionFacetPortletPreferences = new CPOptionFacetPortletPreferences(java.util.Optional.ofNullable(portletPreferences));
```

Comment:
@andrea-ale-sbarra please, either put java.util.Optional in imports or create the Optional value inside the CPOptionFacetPortletPreferences constructor